store/storage.go

```diff
@@ -0,0 +1,8 @@
+package store
+
+type Storage interface {
```

Comment:
I think this interface will be changed for IPFS.

Reply:
I see. I will consider to make change for more general use.

Reply:
I have a question about this code part and comments. In near future, we will use IPFS as a storage. So we are not need to use interface anymore?

Reply:
In the case of using IPFS, we can utilize this interface. IMHO, this kind of interface will be useful when mocking storage for test.